bootstrap/Dockerfile

```diff
@@ -1,6 +1,8 @@
 # TODO(jlewi): How can we make it work with golang:1.8.2-alpine
 FROM golang:1.8.2
 
+ARG KUBEFLOW_REPO=https://github.com/kubeflow/kubeflow.git
```

Comment:
For local builds can we access the files from a volume mount so that we don't have to clone the repo?

Reply:
Now copy code from local repo during stage 1.